### PR TITLE
do not promote self harm

### PR DESCRIPTION
**Pull request by micwoj92**

### CODE REVIEW

**Comment by maximsmol** [comment]
Can you at least fix the grammar

**Comment by micwoj92** [comment]
where

File: linux/srcds_run [hunk 1]
```diff
@@ -429,7 +429,7 @@ quit() {
 		rm -f $PID_FILE
 	fi
 
-	# reset SIGINT and then kill ourselves properly
+	# reset SIGINT and then kill process properly
```
Comment (maximsmol) [inline linux/srcds_run:432]:
```suggestion
	# reset SIGINT and then kill the process properly
```

Comment (micwoj92) [reply linux/srcds_run]: Thank you

**Comment by KingTerrytheTerrible** [comment]
lmfao

**Pull request merged**